src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -1191,4 +1193,81 @@ namespace TerminalAppLocalTests
             VERIFY_IS_TRUE(caughtExpectedException);
         }
     }
+
+    void SettingsTests::TestLayerGlobalsOnRoot()

```

Comment:
What if the user has multiple `"globals"`? Like the following:
```
"globals":{},
"defaultProfile": <val>,
"globals":{}
```
This might be impacted by the other thread I started...

Reply:
this is just gonna be invalid

Reply:
Honestly, I'd kinda just prefer to remove the `"globals"` key in general. Then we'd get rid of all of these 😁

Reply:
me too at this point. However, we haven't ever gotten to the point where we can safely say "we're breaking your settings now" unfortunately. Plus there's that C# tool out there for customizing our settings, and IIRC that tool uses the "globals" object :(